src/components/gltf-model-plus.js

```diff
@@ -595,15 +595,17 @@ class GLTFHubsComponentsExtension {
           // Note: For some reason this was not supported for PDFs. Not sure if it's random or if there is a reason.
           if (shouldUseNewLoader()) {
             if (Object.prototype.hasOwnProperty.call(ext, "link")) {
-              if (["image", "video", "model"].includes(componentName)) {
+            if (["image", "video", "model"].includes(componentName)) {
+                if (!ext.link || !ext.link.href) {
+            console.warn("Warning: Attempted to load a link but the href is missing!", ext);
+            } else {
                 ext["media-link"] = {
-                  src: ext.link.href
-                };
-                delete ext.link;
-              }
-            }
-          }
-
+            src: ext.link.href
+           };
+           delete ext.link;
+                   }
+               }
+           }
```

Comment:
This looks good, but please fix the indentation.

Reply:
Sure, how exact should I fix it?